pyproject.toml

```
@@ -0,0 +1,14 @@
+[tool.isort]
+known_local_folder = ["utils"]
+
+[tool.codespell]
+skip = 'go.sum, **/go.sum, **/third_party, tarantool-*, mage'
```

```suggestion
skip = 'go.sum, **/go.sum, **/third_party, tarantool-*, mage*'
```
We have 2 magefiles, let's skip it all.